src/components/groups/CreateGroup.tsx

```diff
@@ -0,0 +1,185 @@
+      setErrors((prevErrors) => ({
+        ...prevErrors,
+        divisionId: "division level is required",
+      }));
+      return;
+    }
+
+    await createData({
+      name,
+      capacity: capacity as number,
+      telegramLink,
+      divisionId: divisionId as number,
+    }).unwrap();
+    setGroup(initialState);
+    setErrors(initialError);
+  };
+
+  return (
+    <div className="flex flex-col mb-4 items-center w-[100vw] md:w-[80vw] lg:w-[45vw]">
+      <h1 className="text-2xl font-semibold">Create New Group</h1>
+      <form className="w-full px-10" onSubmit={handleSubmit}>
+        <div className="mb-4">
+          <label htmlFor="group-name" className="text-gray-500 mb-2">
+            Group Name
+          </label>
+          <input
+            name="name"
+            type="text"
+            value={group.name}
+            onChange={handleChange}
+            className="outline-none w-full p-1 border rounded leading-tight focus:border-gray-400"
+            placeholder="Enter group name"
+            required
+          />
+          {errors.name && <p className="text-red-500"> {errors.name}</p>}
+        </div>
+        <div className="mb-4">
+          <label htmlFor="group-size" className="text-gray-600 mb-1">
+            Group Size
+          </label>
+          <input
+            id="group-size"
+            name="capacity"
+            type="number"
+            value={group.capacity}
+            onChange={handleChange}
+            className="outline-none w-full p-1 border rounded leading-tight focus:border-gray-400"
+            placeholder="Enter group size"
+            required
+          />
+          {errors.capacity && (
+            <p className="text-red-500"> {errors.capacity}</p>
+          )}
+        </div>
+        <div className="mb-4">
+          <label htmlFor="group-size" className="text-gray-500">
+            Group Telegram Link
+          </label>
+          <input
+            id="telegram link"
+            name="telegramLink"
+            type="text"
+            value={group.telegramLink}
+            onChange={handleChange}
+            className="outline-none w-full p-1 border rounded leading-tight focus:border-gray-400"
+            placeholder="Enter group telegram link"
+            required
+          />
+          {errors.telegramLink && (
+            <p className="text-red-500"> {errors.telegramLink}</p>
+          )}
+        </div>
+        <div className="mb-4">
+          <label htmlFor="division-level" className="text-gray-500 ">
+            Select Division
+          </label>
+          <select
+            id="division-level"
+            name="divisionId"
+            value={group.divisionId}
+            onChange={handleChange}
+            className="outline-none w-full p-1 border rounded leading-tight focus:border-gray-400"
+            required
+          >
+            <option value="" className="text-sm sm:text-base md:text-lg">
+              -select-
```

Comment:
It is recommended to use `--select--` with double dashes, as it is a more commonly used convention for indicating a placeholder